src/generic_ui/polymer/copypaste.ts

```diff
@@ -22,7 +28,9 @@ Polymer({
       this.startGetting();
     }
   },
+  lastState: STATE.SHARING,
```

Comment:
Yup.  Well, mostly.  I picked it because I am about a percent more confident that I did not miss anything with the getting cases so, if by some chance I screwed up, I think this will cover it better.

But, yes, in theory, arbitrarily-chosen default.